pkgs/servers/sql/postgresql/generic.nix

```diff
@@ -147,7 +151,11 @@ let
       ++ lib.optionals gssSupport [ "--with-gssapi" ]
       ++ lib.optionals pythonSupport [ "--with-python" ]
       ++ lib.optionals jitSupport [ "--with-llvm" ]
-      ++ lib.optionals stdenv'.hostPlatform.isLinux [ "--with-pam" ]
+      # Building with linux-pam in pkgsStatic gives a few "undefined reference" errors like this:
+      #   /nix/store/3s55icpsbc36sgn7sa8q3qq4z6al6rlr-linux-pam-static-x86_64-unknown-linux-musl-1.6.1/lib/libpam.a(pam_audit.o):
+      #     in function `pam_modutil_audit_write':(.text+0x571):
+      #     undefined reference to `audit_close'
+      ++ lib.optionals (stdenv'.hostPlatform.isLinux && !stdenv'.hostPlatform.isStatic) [ "--with-pam" ]
```

Comment:
This probably needs the following in its .pc file (upstream), if you're interested in fixing it:

```
Requires.private: audit
```

Reply:
Given it's autotools, something like this: https://github.com/rockdaboot/libpsl/pull/246

Reply:
Yes, that should do it.

Reply:
Work at all or work with postgres?